src/ports/postgres/modules/graph/sssp.py_in

```diff
@@ -432,11 +433,17 @@ def graph_sssp(schema_madlib, vertex_table, vertex_id, edge_table,
 								SELECT 1
 								FROM {oldupdate} as oldupdate
 								WHERE {checkg_oo_sub}
-								);
-						DROP TABLE {out_table};
-						ALTER TABLE {temp_table} RENAME TO {out_table};"""
-
-				plpy.execute(sql_del.format(**locals()))
+								);"""
+					plpy.execute(sql_del.format(**locals()))
+					sql_del = "DROP TABLE {out_table}"
+					plpy.execute(sql_del.format(**locals()))
+					sql_del = "ALTER TABLE {temp_table} RENAME TO {out_table};"
+					plpy.execute(sql_del.format(**locals()))
```

Comment:
Same as previous comment - all these `plpy.execute` can directly run the string, simplifying the code.